tryalgo/sudoku.py

```diff
@@ -17,22 +17,29 @@
 # sets
 def assignation(r, c, v): return r * N4 + c * N2 + v
 
+
```

Comment:
Ah yeah, for this file we chose not to have double lines, because of the printed version.

Reply:
ok let's leave double lines. I will correct it back.